es6-destructuring/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title></title>
+  </head>
+  <body>
+    <script>
+      const book1 = {
+        title: 'Goodnight Punpun',
+        author: 'Inio Asano',
+        libraryID: 3353,
+      };
+
+      const { title, author, libraryID } = book1;
+
+      console.log(
+        `The title of the book is ${title}, the author is ${author}, and the libary id is ${libraryID}`
+      );
+
+      const book2 = {
+        title2: 'Les Fleurs de mal',
+        author2: 'Charles Baudelaire',
+        libraryID2: 2345,
+      };
+      const { title2, author2, libraryID2 } = book2;
```

Comment:
This isn't the desired solution. You should do this without renaming the keys in `book2`. You need to use the destructuring syntax that lets you rename the variable that will be created.